distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -735,4 +738,38 @@ public void clearCatalog() {
     public void clearCache() {
         console.runCommand(CLEAR_CACHE);
     }
+
+    public ValidatableResponse executeOpenSearch(String format, String... query) {
+        StringBuilder buffer = new StringBuilder(OPENSEARCH_PATH.getUrl()).append("?")
+                .append("format=")
+                .append(format);
+
+        for (String term : query) {
```

Comment:
Could be refactored to be `            Arrays.stream(query)
                    .forEach(term -> buffer.append("&").append(term));`